proposals/0352-implicit-open-existentials.md

```diff
@@ -4,6 +4,7 @@
 * Authors: [Doug Gregor](https://github.com/DougGregor)
 * Review Manager: [Joe Groff](https://github.com/jckarter)
 * Status: **Implemented (Swift 5.7)**
+* Upcoming Feature Flag: `ImplicitOpenExistentials` (Implemented in Swift 5.8) (Enabled in Swift 6 language mode)
```

Comment:
```suggestion
* Upcoming Feature Flag: `ImplicitOpenExistentials` (Implemented in Swift 6.0) (Enabled in Swift 6 language mode)
```

The upcoming feature flag's behavior was only implemented recently (for Swift 6.0), https://github.com/apple/swift/pull/72018